apps/backoffice-v2/src/common/hooks/useHomeLogic/useHomeLogic.ts

```diff
@@ -16,6 +16,10 @@ export const HomeSearchSchema = z.object({
   mmTo: z.string().date().optional(),
   casesFrom: z.string().date().optional(),
   casesTo: z.string().date().optional(),
+  allowAllDates: z
+    .string()
+    .transform(value => (value === 'true' ? true : false))
```

Comment:
```suggestion
    .transform(value => value.toLowerCase() === 'true')
```

also, might we have already implemented `string -> boolean` somewhere common?